mappings/net/minecraft/util/registry/RegistryKey.mapping

```diff
@@ -0,0 +1,16 @@
+CLASS net/minecraft/class_5321 net/minecraft/util/registry/RegistryKey
```

Comment:
I really want to use RegistryIdentifier instead, since at the end of the day, it is still an identifier

Reply:
It's not an identifier though, it does not extend the identifier class. It's a registry id + identifier pair.

Reply:
good point, however it is used for finding the entry, it is almost like it is an identifier for an entry. I am happy with RegistryKey tho. just concerned about the consistency of it since identifier is technically a key as well if you put it in the same context

Reply:
Yeah, tbh this class is a little confusing, as the registry id part of it doesn't seem to be used anywhere. However it's useful to have this distinction so we can have `containsId` vs `containsKey` and the likes.

Reply:
that sounds fair enough

Reply:
This is more analogous to that object which is a compound of two identifier for locating an atlas and then a sprite in the atlas. I forgot what that class is named, might be a good idea to make two have similar names.